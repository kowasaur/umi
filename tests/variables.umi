void main() {
    string message = "hello!";
    print(message);

    int32 number = 32;
    int32 number_2 = number;
    printn(number);
    printn(number_2);

    message = "goodbye :(";
    print(message);
}
