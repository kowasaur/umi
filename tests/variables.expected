hello!
32
32
goodbye :(
